client/index.html

```diff
@@ -317,24 +329,24 @@ <h5>Already Installed Bots</h5>
                 </tr>
               </thead>
               <tbody>
-                <tr v-for="b in importing.installedBots">
+                <tr v-for="bot in importing.installedBots">
                   <td>
-                    <label v-if="!b.isIdentical">
-                      <input v-model="b.doImport" class="filled-in" id="enable1" name="enable1" type="checkbox" />
+                    <label v-if="!bot.isIdentical">
+                      <input v-model="bot.doImport" class="filled-in" type="checkbox" />
                       <span></span>
                     </label>
                     <span class="red white-text" v-else>
                       (exists in import)
                     </span>
                   </td>
-                  <td>{{ b.name }}</td>
-                  <td><code>{{ b.responds_to }}</code></td>
-                  <td><a :href="b.url"><code>{{ b.url }}</code></a></td>
+                  <td>{{ bot.name }}</td>
+                  <td><code>{{ bot.responds_to }}</code></td>
+                  <td><a :href="bot.url"><code>{{ bot.url }}</code></a></td>
                 </tr>
               </tbody>
             </table>
 
-            <h5>Importing Bots</h5>
+            <h5>Imported Bots</h5>
```

Comment:
This implies the bot has already been imported, hence why I think I used "Importing". Maybe "New Bots" or "Bots to Import"?

Reply:
New bots is probably better.

Reply:
Done.